dataclients/kubernetes/definitions/testdata/validation/route-with-duplicate-hosts.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: zalando.org/v1
```

Comment:
I think the test fixture for Ingress is missing

Reply:
Right now we don't validate ingress in skipper we only do in ValidationWebhook, I think we will need another PR to migrate current ingress adhock validation to validator object.

Reply:
What I mean by adhock validation https://github.com/zalando/skipper/blob/master/dataclients/kubernetes/ingress.go#L257C1-L269C2

Reply:
https://github.com/zalando/skipper/pull/2757